web.browser/src/containers/PostList/PostListContainer.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PostList from './PostList';
+import PostToolBar from '../../components/PostToolBar'
+import Chip from 'material-ui/Chip';
+import { updateVote, sortByDate, sortByUpvotes } from '../../redux/actions';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+
+
+
+
+class PostListContainer extends Component {
+
+    constructor(){
+        super();
+        this.updateVote = this.updateVote.bind(this);
+        this.sortByDate = this.sortByDate.bind(this);
+        this.sortByUpvotes = this.sortByUpvotes.bind(this);
+    }
+
+    sortByDate() {
+        this.props.dispatch(sortByDate());
+    }
+
+    sortByUpvotes(){
+        this.props.dispatch(sortByUpvotes());
+    }
+
+    updateVote( postId ){
+        if ( postId ) this.props.dispatch(updateVote(postId));
+    }
+
+    categoryChips( category ){
+        return <Chip key="category" style={{ position:'absolute', right: '50px', bottom: '20px' }} >{ category }</Chip>
```

Comment:
Pretty long line here. Either move the style object to a variable or place on multiple lines.